pom.xml

```diff
@@ -117,6 +117,11 @@
             <version>2.2.2</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
```

Comment:
Where liquibase version is set? I don't see properties block.

Reply:
I used the version from "spring-boot-dependencies-3.0.3.pom".